apps/catalogue/src/components/CohortCards.vue

```diff
@@ -8,12 +8,21 @@
     </RouterLink>
     <h1 class="bg-white">{{ network }} cohorts</h1>
     <p>This page lists all cohorts partner in the {{ network }} network.</p>
-    <InputSearch v-model="searchTerms" placeholder="search cohorts" />
+    <InputSearch
+      id="cohort-cards-search-input"
+      v-model="searchTerms"
+      placeholder="search cohorts"
+    />
     <p>Found {{ count }} cohorts.</p>
     <div class="row">
       <div
-        class="col-xl-3 col-lg-4 col-md-6 col-sm-12 mb-4 d-flex align-items-stretch"
```

Comment:
damn the formatting changes :-) how come do we now. Are we using different formatters maybe?